man/systemd-nspawn.xml

```diff
@@ -1363,9 +1363,11 @@ After=sys-subsystem-net-devices-ens1.device</programlisting>
         multiple times for creating multiple independent bind mount points.</para>
 
         <para>Mount options are comma-separated. <option>rbind</option> and <option>norbind</option> control whether
-        to create a recursive or a regular bind mount. Defaults to "rbind". <option>idmap</option> and <option>noidmap</option>
-        control if the bind mount should use filesystem id mappings. Using this option requires support by the source filesystem
-        for id mappings. Defaults to "noidmap".</para>
+        to create a recursive or a regular bind mount. Defaults to "rbind". <option>idmap</option>, <option>noidmap</option>,
+        and <option>rootidmap</option> control if and how the bind mount should use filesystem id mappings. Using this option
+        requires support by the source filesystem for id mappings. <option>idmap</option> enables filesystem id mapping,
+        <option>noidmap</option> disables it, and <option>rootidmap</option> maps the owner of the bind-mounted directory to
+        the root user within the container. Defaults to "noidmap".</para>
```

Comment:
if we add a third option here we should probably explain this better, i.e. say the net effect of each option, maybe something like this:

> With `x` being the container's UID range offset, and `y` being the length of the container's UID range, and `p` being the owner UID of the bind mount source inode on the host:
>
> 1. If `idmap` is used, any user `z` in the UID range `0 … y` as seen from inside the container is mapped to the same `z` in the same `0 … y` range on the host. All host users outside of that range are mapped to `nobody` inside the container.
>
> 2. If `noidmap` is used, any user `z` in the range `0 … y` seen from inside of the container is mapped to `x + z` in the `x … x + y` range on the host. All host users outside of that range are mapped to `nobody` inside the container.
>
> 3. If `rootidmap` is used, any user `z` in the range `1 … y` seen from inside of the container is mapped to `x + z` in the `x + 1 … x + y` range. The user `0` seen from inside of the container is mapped to `p` on the host. Al users outside of `x + 1 … x + y` and not equal to `p` on the host are mapped to `nobody` inside the container.